index.pug

```diff
@@ -0,0 +1,96 @@
+html
+  head
+    link(rel="stylesheet" href="/style.css")
+    script(src="../cars.js")
+    body
+     h1.🚦MY CAR COLLECTION API 🏎
+
+     h2. here's a curated list of cars desired by customers /people
+     
+    ul.list 
+      p Get the cars listed 
+
+      li.carList
+        p 
+          strong GET 
+          | /allCars
+        div#getAllCars
+            p Method: GET 
+            p Route: http://localhost:2035/cars/allCars   
+            p Desired outcome : this should bring out all the cars 
+
+      li.listItems 
+            p 
+          strong GET 
+          |/all Cars By Id
+        div#getById
+            p method : GET 
+            p route: http.//localhost:7825/cars/allCars{id}
+            p these are all the cars by their id 
+        p 
+          strong POST 
+          | /allCars
+        div 
+            p Method: POST 
+            p Route: http://localhost2035/cars/allCars
+            p Headers: content type : application/JSON
+            p Body: {model, type, year, modelname, founder}
+            p Desired Outcome ; cerates new fileds with list of new cars. 
+
+            li.listCars
+        p 
+          strong DELETE 
+          | /car By Id
+        div 
+            p Method: DELETE 
+            p Route: http://localhost2035/cars/CarsById
+            p Headers:   content type : application/JSON
+            p Desired outcome : Delets cars by Id provided 
+
+      li.listItems 
+        p 
+          strong Patch 
+          | /Cars By Id
+        div 
+            p Method: Patch
+            p Route: http://localhost2035/cookies/CarsById
+            p Headers: Content type :application/JSON
+            p Body: {model, type, year, modelname, founder}
+            p Desired outcome : Cars provided with a  specific id will be updated.
+            
+      li.listItems 
+        p 
+          strong GET 
+          | /BagsByFounder
+        div 
+            p Method: GET .
+            p Route: http://localhost2035/bags/carByFounder.
+            p Headers:   content type : application/JSON.
+            p Desired outcome : gets the cars created by the founder.
+      li.listItems 
+        p 
+          strong GET 
+          | /bagsByModel
+        div 
+            p Method: GET .
+            p Route: http://localhost2035/bags/carByModel.
+            p Headers: content type : application/JSON.  
+            p Desired outcome :Gets all cars with the model provided .
+      li.listItems 
+        p 
+          strong GET 
+          | /bagsByModelName
+        div 
+              p Method: GET. 
+              p Route: http://localhost2035/cars/carByModelName.
+              p Headers: content type : application/JSON .  
+              p Desired outcome : gets the cars with the modelName provided .
+        li.listItems 
```

Comment:
These feel like weird routes and also don't tell me how I'd provide the founder or model. I'd expect `/cars/altima` to get me any of those cars. This would mean all of these are the same route where I could say `/cars/toyota` or `/cars/senna` and it would return the cars. This would mean it is the same route just like how we did title and director in Movies API. If you want them to be separate routes, the REST way to do that would be `/cars/model/toyota` vs `/cars/modelname/senna`